### DISCUSSION TITLE
PointerRelease Event on RadioButton don't fire

### DISCUSSION THREAD

**Discussion by Tenjim** [AvaloniaUI/Avalonia#8853] [Q&A] [answered]
Hi guys, 

I currently use behaviors from [here](https://github.com/wieslawsoltes/AvaloniaBehaviors)

I have a radiobutton like this  : 

```
<RadioButton x:Name="TestRadio" >
    <Interaction.Behaviors>
        <EventTriggerBehavior EventName="PointerReleased">
            <InvokeCommandAction Command="{Binding CreateCommand}" />
        </EventTriggerBehavior>
    </Interaction.Behaviors>
</RadioButton>
```

This event PointerReleased is not trigger, Only Click Event work on this case. Is it by design of it's come to xaml behaviors ?

Thanks in advance.

**Comment by maxkatz6** [comment, marked as answer]
By design. PointerReleased is handled by the control itself.
@wieslawsoltes I think it would be useful to allow to specify handling strategy (bubble, tunnel, direct).

**Reply by wieslawsoltes** [reply]
> @wieslawsoltes I think it would be useful to allow to specify handling strategy (bubble, tunnel, direct).

@maxkatz6 Not for EventTriggerBehavior as it's also for clr events. There is specialised version with that functionality called RoutedEventTriggerBehavior

https://github.com/wieslawsoltes/AvaloniaBehaviors/blob/2f5e49c050bcb57a03e26689eb9e8eef987f3bab/src/Avalonia.Xaml.Interactions/Custom/RoutedEventTriggerBehavior.cs#L11

**Reply by wieslawsoltes** [reply]
Also RoutedEventTriggerBehavior does not use reflection and it works nicely with Xaml editor (Rider)

**Comment by wieslawsoltes** [comment]
Also there is custom behavior for implementing event handlers called PointerEventsBehavior

https://github.com/wieslawsoltes/AvaloniaBehaviors/blob/2f5e49c050bcb57a03e26689eb9e8eef987f3bab/src/Avalonia.Xaml.Interactions/Events/PointerEventsBehavior.cs#L71-L90

There is bunch of other custom behaviours for events https://github.com/wieslawsoltes/AvaloniaBehaviors/tree/master/src/Avalonia.Xaml.Interactions/Events

**Comment by Tenjim** [comment]
Thank guys, I'm checking this :)
